CHANGELOG.md

```diff
@@ -15,6 +15,58 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 
 ### Fixed
 
+## [1.9.67] (https://github.com/rainbow-me/rainbow/releases/tag/v1.9.67)
+
+### Added
+
+- Add tracking of how often "Insufficient ETH" error message is returned (#6590)
+- Prince of hill (#6575)
+- New Maestro E2E CI pipeline (#6478)
+- Wire up market stats on the expanded state for Rainbow-created tokens (#6597)
+- Add tracking for wallet screen pull to refresh (#6608)
+- Add clearKeychain for Maestro tests (#6600)
+- Implement subscript notation for charts with more than 8 magnitude decimals (#6558)
+- Add Token Creator to Expanded State (#6612)
+
+### Changed
+
+- Remove app store review pre-prompt & ask more often (#6569)
+- Changed the RainbowError constructor to be slightly different than the normal Error constructor (#6557)
+- RecyclerListView cleanup (#6521)
+- Adjust query store's internal set usage to cover all enabled changes (#6591)
+- Changed height of pkey box (#6585)
+- Convert Navigation Stack + ExplainSheet to Typescript (#6566)
+- Update the Readme to specify the version of Node required (#6601)
+- Update AndroidManifest.xml (#6598)
+- Bump react-native-keychain from 8.0.0 -> 9.2.3 (#6592)
```

Comment:
we actually probably should change this to 10.0.0 if we ended up there

Reply:
good catch